pyscriptjs/tests/test_examples.py

```diff
@@ -0,0 +1,85 @@
+"""Each example requires the same three tests:
+
+- Test that the initial markup loads properly (currently done by testing the <title>
+  tag's content)
+- Testing that pyodide is loading properly
+- Testing that the page contains appropriate content after rendering
+
+The single function iterates through the examples, instantiates one playwright browser
+session per example, and runs all three of each example's tests in that same browser
+session.
+ """
+
+import math
+import re
+import time
+from urllib.parse import urljoin
+
+from playwright.sync_api import sync_playwright
+
+
+def test_examples(config):
+    # Set up time/iterations to allow for testing
+    max_test_time = config["default_values"]["MAX_TEST_TIME"]  # e.g. 30 seconds
+    test_time_increment = config["default_values"][
+        "TEST_TIME_INCREMENT"
+    ]  # e.g. 1/4 second
+    test_iterations = math.ceil(max_test_time / test_time_increment)  # 120 iterations
+
+    for example in config["examples"]:
```

Comment:
the not so great thing about this pattern is that this will show as one single test and will fail all tests as soon as the first test fails, not giving my granularity into what tests are failing, etc... Using [parametrized tests](https://docs.pytest.org/en/stable/how-to/parametrize.html) seems like a good fit for this use case